db/migrate/20150122082200_add_scopes_to_oauth_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopesToOauthApplications < ActiveRecord::Migration
```

Comment:
I'm not sure I understand why we need this. According to your link, this is only relevant if we add `enable_application_scopes`, which we're not doing - and I don't see where we are actually using application-level scopes anyway. What am I missing?